docs/configuring-playbook-bridge-mx-bridge-skype.md

```diff
@@ -0,0 +1,36 @@
+# Setting up MX Puppet Skype (optional)
+
+The playbook can install and configure
+[mx-puppet-skype](https://github.com/Sorunome/mx-puppet-skype) for you.
+
+See the project's
+[documentation](https://github.com/tulir/mautrix-hangouts/wiki#usage) to learn
+what it does and why it might be useful to you.
+
+To enable the [Skype](https://www.skype.com/) bridge just use the following
+playbook configuration:
+
+
+```yaml
+matrix_mx_puppet_skype_enabled: true
+
+# You can put any string here, but generating a strong one is preferred (e.g. `pwgen -s 64 1`)
+matrix_mx_puppet_skype_appservice_token: ""
```

Comment:
It would be simpler if you move this to `group_vars/matrix_servers` and derive it from `matrix_synapse_macaroon_secret_key`, like we do for the other bridges.

This way, we won't need to ask people to define these values manually. All they would need is to set `matrix_mx_puppet_skype_enabled: true` and that's it.

Just create a new `# mx-puppet-skype` section somewhere in `group_vars/matrix_servers` and do like we do for the other bridges in that file.

Besides defining `matrix_mx_puppet_skype_appservice_token` and `matrix_mx_puppet_skype_homeserver_token` there, you can add `'matrix-synapse.service'` to `matrix_mx_puppet_skype_systemd_required_services_list` from `group_vars/matrix_servers` as well. See my other review comment for more about this.

Reply:
Fixed!